excercsises/4_inheritance/home.spec.js

```diff
@@ -10,15 +10,14 @@ describe("Protractor Workshop app", function () {
   });
 
   it('should have "Example headline 2" carousel item after clicking on next arrow', function () {
-    const expectedHeader = "Example Headline 2";
-    homePage.caruselButton();
-    expect(homePage.getHeadline()).toEqual(expectedHeader);
+    let  headLine = "Example Headline 2";
+    expect(homePage.checkHeadLine()).toEqual(headLine);
   });
 
   it("should display drop down after clicking on About menu item", function () {
-    homePage.dropdownMenu();
+    homePage.checkDropDownMenuWorks()
```

Comment:
Is this method really checks the dropdown menu works? 
Why in this method you also click the History?